leap_c/trainer.py

```diff
@@ -322,6 +322,12 @@ def run(self) -> float:
         """Call this function in your script to start the training loop."""
         self.to(self.device)
 
+        for optimizer in self.optimizers:
+            for state in optimizer.state.values():
+                for k, v in state.items():
+                    if isinstance(v, torch.Tensor):
+                        state[k] = v.to(self.device)
+
```

Comment:
@JasperHoffmann I noticed a problem when I was continuing the training, basically loading the model and the optimizers _then_ moving the model to the device causes the optimizers' parameters to stay on CPU. This is just a temporary workaround.